### ISSUE TITLE
document how to install openebs via helm. 

### ISSUE THREAD

**Issue by kmova** [kontena/pharos-docs#187]
The OpenEBS page (https://pharos.sh/docs/addons/openebs.html#) says it is depreciated.  Can I raise a PR that updates this page with providing instructions on how to set this up via the helm stable version?

There are some openebs users who would like to install openebs via the Kontena Pharos.

**Comment by jakolehm** [comment]
Yes, we accept pull requests 😄 ... the easiest way to enable openebs is probably via helm addon, see: https://www.pharos.sh/docs/addons/helm.html#configuration

**Comment by Edke** [comment]
hi @jakolehm and thank you @kmova for an effort to help me with OpenEBS on Pharos.

I can definitely help you to add more info or test OpenEBS as addon while creating new cluster.
